polyply/src/gen_itp.py

```diff
@@ -20,7 +20,13 @@
 import vermouth
 import vermouth.forcefield
 from vermouth.log_helpers import StyleAdapter, get_logger
-from vermouth.file_writer import open, DeferredFileWriter
+# patch to get rid of martinize dependency
+try:
+    from vermouth.file_writer import deferred_open
+except ImportError:
+    from vermouth.file_writer import open
+    deferred_open = open
+from vermouth.file_writer import DeferredFileWriter
```

Comment:
You don't use the `deferred_open` at the moment.

Reply:
indeed I forgot to push the last commit.